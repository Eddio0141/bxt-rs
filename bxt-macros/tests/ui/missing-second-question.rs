use bxt_macros::pattern;

fn main() {
    pattern!(00 ? 00);
}
