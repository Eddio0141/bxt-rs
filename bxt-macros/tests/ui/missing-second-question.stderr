error: missing second `?`
 --> $DIR/missing-second-question.rs:4:17
  |
4 |     pattern!(00 ? 00);
  |                 ^
